### PR TITLE
Fix hydration errors

### PR DESCRIPTION
**Pull request by cesarvarela**
To get this PR working on staging (and production), a few changes need to be made on Netlify:

Add 

```
NODE_VERSION=18
NODE_OPTIONS=--max-old-space-size=4096
```

Remove:
```
NPM_FLAGS
NPM_VERSION
```

PR is working here: https://github.com/cesarvarela/aiid/pull/38

### CODE REVIEW

**Comment by pdcp1** [comment]
@cesarvarela If this PR is ready to review please remove the Draft

File: site/gatsby-site/plugins/gatsby-theme-i18n-react-i18next/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// noop
```
Comment (pdcp1) [inline site/gatsby-site/plugins/gatsby-theme-i18n-react-i18next/index.js:1]: Is this needed?

**Review by pdcp1 — CHANGES_REQUESTED** [review]

File: site/gatsby-site/plugins/gatsby-theme-i18n-react-i18next/index.js (hunk 1 above)
Comment (cesarvarela) [reply site/gatsby-site/plugins/gatsby-theme-i18n-react-i18next/index.js:1]: Looks like it is not, but many gatsby packages do that so 🤷

Comment (pdcp1) [reply site/gatsby-site/plugins/gatsby-theme-i18n-react-i18next/index.js:1]: let's keep it as it is then 👍

**Review by pdcp1 — APPROVED** [review]

**Comment by pdcp1** [comment]
@cesarvarela This is approved and ready to merge as soon the tests passed 👍

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**